package.json

```diff
@@ -7,10 +7,11 @@
     "build": "tsc -p src/server/tsconfig.json && tsc -p src/client/tsconfig.json",
     "start": "npm run build && node ./lib/server/index.js",
     "start-debug": "cross-env DEBUG=stackoverflowtags:* npm run start",
-    "lint": "eslint ./test ./src/server/index.js",
+    "lint": "eslint ./test/test.js ./src/server/index.js",
```

Comment:
Please check all test files and enable [eslint-plugin-testcafe](https://www.npmjs.com/package/eslint-plugin-testcafe) if necessary.